src/operators/less_than.cpp

```diff
@@ -31,6 +31,48 @@
 #include "../bigint.hpp"
 
 namespace libbig {
+  bool largeInt:: operator < (const largeInt& z) {
+    // sign = 0: positive number
+    // sign = 1: negative number
+    // flip if its reverse
 
+    int NEGATIVE = 1;
+    int POSITIVE = 0;
```

Comment:
It's the opposite. If `this->sign` (or, in other words, if `this->sign == 1`), the number is positive. This is stated in [src/operators/ostream_operator.cpp](https://github.com/ayaankhan98/bigINT/blob/main/src/operators/ostream_operator.cpp) : `if (!z.sign) { out << '-'; }`.
```suggestion
    // sign = 0: negative number
    // sign = 1: positive number
    // flip if its reverse

    int POSITIVE = 1;
    int NEGATIVE= 0;
```